sample/src/main/AndroidManifest.xml

```diff
@@ -17,128 +17,128 @@
       >
     <activity
         android:name=".SomeViewsWithDifferentVisibilitiesActivity"
-        android:exported="false"
+        android:exported="true"
```

Comment:
Hi @alorma ! Just a question. Why do we need to set this as `true`? It's something related to passing the tests?

Reply:
just trying things... tests are not passing on android 12 and i'm investigating why